Flora/Flora/View Controllers/PostCell.m

```diff
@@ -58,6 +59,23 @@ - (void)setPost:(Post *)post {
     self.likeCountLabel.text = [NSString stringWithFormat:@"%@ likes", [self.post.likeCount stringValue]];
     self.commentCountLabel.text = [NSString stringWithFormat:@"%@ comments", [self.post.commentCount stringValue]];
     
+    [self updateLikes];
+    
+    
+}
+
+- (void) updateLikes {
+    self.likeCountLabel.text = [NSString stringWithFormat:@"%lu likes", self.post.userLikes.count];
+
+    if ([self.post.userLikes containsObject:[PFUser currentUser].username]) {
+        [self.likeButton setImage:[UIImage systemImageNamed:@"heart.fill"] forState:UIControlStateNormal];
+        [self.likeButton setTintColor:[UIColor redColor]];
+        [self.likeButton setUserInteractionEnabled:YES];
```

Comment:
move this back to the same method that disables it.

Reply:
implemented feedback in later branch